RELEASE.md

```diff
@@ -41,38 +42,59 @@ The [Draft New Release](./.github/workflows/draft-release.yml) workflow is the f
 It must be triggered manually using the [Actions](https://github.com/SeldonIO/seldon-core/actions/workflows/draft-release.yml) interface in GitHub UI.
 
 When triggering the workflow, you must:
-- select the release branch (here `release-0.1`)
-- specify the release `version` (here `v0.1.0-rc1`).
+- Select the release branch (here `release-0.1`)
+- Specify the release `version` (here `v0.1.0-rc1`).
 
 ![Triggering Draft Workflow](.images/release-4.png)
 
-This workflow cannot run on `master` branch.
-It will also validate provided `version` against SemVer regex.
+This workflow cannot run on the `v2` branch.
+
+It will validate the provided `version` against a SemVer regex.
 
 It will create a few commits with:
-- updated helm charts
-- updated k8s yaml manifests
-- updated changelog
+- Updated Helm charts
+- Updated Kubernetes YAML manifests
+- An updated changelog
 
 ![Created Commits](.images/release-3.png)
 
 Once the workflow finishes, you will find a new release draft waiting to be published.
 
 ![Draft Release](.images/release-5.png)
 
-> :warning: **NOTE:** Before publishing the release run the images build workflow and necessary tests (not yet automated)!
+> :warning: **NOTE:** Before publishing the release, run the images build workflow and necessary tests (not yet automated)!
 
 
 ### Build docker images Action
 
 The [Build docker images](./.github/workflows/images.yml) workflow is the second one to run.
 
-It must be triggered manually using the [Actions](https://github.com/SeldonIO/seldon-core/actions/workflows/draft-release.yml) interface in GitHub UI.
+It must be triggered manually using the [Actions](https://github.com/SeldonIO/seldon-core/actions/workflows/draft-release.yml) interface in the GitHub UI.
 
 When triggering the workflow, you must:
-- select the release branch (here `release-0.1`)
-- specify the release `version` (here `0.1.0-rc1` - note lack of `v` prefix!).
+- Select the release branch (here `release-0.1`)
+- Specify the release `version` (here `0.1.0-rc1` - note lack of `v` prefix!).
 
 ![Triggering Build images](.images/release-6.png)
 
 This workflow will then run unit tests and build a series of Docker images that will be automatically pushed to [DockerHub](https://hub.docker.com/).
+
+
+### Add Go module tags
```

Comment:
Could we add required command to run (unless we'd want to automate this via GitHub action).

Reply:
Was already on this -- that's why I moved back to draft!  Great minds think alike :wink:

Reply:
:) - thought it will be something simpler like
```bash
TAG=v2.0.0
git tag apis/go/${TAG}
git tag components/tls/${TAG}
...
```

Reply:
I like to describe processes -- if it's completely defined, it should just be a script and the instruction would be "run X"